### DISCUSSION TITLE
Is it possible to add labels to PRs?

### DISCUSSION THREAD

**Discussion by jack-evans** [lindell/multi-gitter#249] [Q&A] [answered]
I want to be able to tag PR's with like `automation` etc so I can then have further automation that once the PRs are approved by devs they can be merged in gradually. Is there a way to do this with multi-gitter?

**Comment by lindell** [comment]
That option does currently not exist. But it should be easy to implement. Which platform are you using? (GitHub?) I would be happy to merge a PR for it.

**Reply by jack-evans** [reply]
Yes GitHub! Not sure if there are any equivalents for gitlab etc...

**Reply by jack-evans** [reply]
Happy to take a look if you're looking for contributors but I'll have to try and get my head around the code base first and see what I can do 😊

**Reply by lindell** [reply]
I think this PR is a decent example of adding a new flag that basically only passing the request down.

https://github.com/lindell/multi-gitter/pull/232/files

It would be nice to support all SCMs, butnkust GitHub is fine as well.

**Reply by jack-evans** [reply]
Had a brief look, looks like with the GitHub API you can't add labels to a Pr on create... you then have to call the issues API to update the PR with the labels! Never as easy as it seems 🤬 will see what I can put together though

**Reply by lindell** [reply]
"Never as easy as it seems"
Indeed that's 99% of projects 😅

It's the same thing with reviewers and assignees FYI.

**Reply by jack-evans** [reply]
made an initial stab here https://github.com/lindell/multi-gitter/compare/master...jack-evans:labels-for-pr?expand=1 feel free to let me know if there is anything obviously wrong :)

**Reply by lindell** [reply]
It looks good :) 
Just two thing. The readme does not have to be updated since it will automatically be updated based on the description of the flag, and please add a table test.

**Reply by jack-evans** [reply]
Apologies lost track of this, will rebase and try getting a Pr in soon!

**Comment by lindell** [comment, marked as answer]
This has now been added in https://github.com/lindell/multi-gitter/releases/tag/v0.43.0 😄
